pynitrokey/nk3/device.py

```diff
@@ -83,11 +84,8 @@ def reboot(self, mode: BootMode = BootMode.FIRMWARE) -> None:
             elif mode == BootMode.BOOTROM:
                 self._call(Command.UPDATE)
         except OSError as e:
-            if e.errno == errno.EIO:
-                # IO error is expected as the device does not respond during the reboot
-                pass
-            else:
-                raise e
+            # OS error is expected as the device does not respond during the reboot
+            self.logger.debug("ignoring OSError after reboot", exc_info=e)
```

Comment:
1. Nit: here `self.logger.exception("ignoring OSError after reboot")` would be more semantic, and AFAIK should be the equivalent of the used call.
2. In general one should check the actual error, but I understand you skipped that in the name of simplicity for multi-platform support.

Reply:
1. I think the difference is the error level.  `exception` uses the `ERROR` level, but I think that would be misleading because it does not indicate an error but expected behavior.
2. Exactly.  We could introduce different cases for different operating systems, but that would probably be pretty hard to maintain.

Reply:
OK!